### PR TITLE
Multi agent models update

### PR DESCRIPTION
**Pull request by annabellasd**

### CODE REVIEW

**Review by htrangtr — CHANGES_REQUESTED** [review]
I tried to be as detailed as I could! I checked that:
- all unicodes are converted to text
- all outputs look good
- codes and comments are aligned so that it's easy to read.

I also checked to see if we can combine any functions in this set of lecture notes (aside from the lake model), but I couldn't find any. Since it's subjective, it would be great if you could go through them too.

File: lectures/multi_agent_models/arellano.md [hunk 1]
```diff
@@ -312,56 +312,55 @@ using Test
...
 function one_step_update!(ae,
                           EV,
                           EVd,
                           EVc)
 
     # unpack stuff
-    @unpack β, γ, r, ρ, η, θ, ny, nB = ae
-    @unpack ygrid, ydefgrid, Bgrid, Π, vf, vd, vc, policy, q, defprob = ae
+    (;beta, gamma, r, rho, eta, theta, ny, nB) = ae
+    (;ygrid, ydefgrid, Bgrid, Pi, vf, vd, vc, policy, q, defprob) = ae
```
[the remark this answers is not in the document]
Comment (annabellasd) [reply lectures/multi_agent_models/arellano.md:355]: no idea, maybe readability?

File: lectures/multi_agent_models/lucas_model.md [hunk 2]
```diff
@@ -393,39 +393,39 @@ using Distributions, Interpolations, LaTeXStrings, Parameters, Plots, Random
...
-    # set h(y) = β * int u'(G(y,z)) G(y,z) ϕ(dz)
+    # set h(y) = beta * int u'(G(y,z)) G(y,z) phi(dz)
     h = similar(grid)
     for (i, y) in enumerate(grid)
-        h[i] = β * mean((y^α .* shocks).^(1 - γ))
+        h[i] = beta * mean((y^alpha .* shocks).^(1 - gamma))
     end
 
-    return (γ = γ, β = β, α = α, σ = σ, ϕ = ϕ, grid = grid, shocks = shocks, h = h)
+    return (gamma = gamma, beta = beta, alpha = alpha, sigma = sigma, phi = phi, grid = grid, shocks = shocks, h = h)
```
[the remark this answers is not in the document]
Comment (annabellasd) [reply lectures/multi_agent_models/lucas_model.md:416]: yes thanks!

**Comment by htrangtr** [comment]
Aside from `≉` symbol in schelling.md, everything else looks good

**Comment by htrangtr** [comment]
Output error in Optimal Taxation without State-Contingent Debt - messaged you! Everything else looks good!

**Pull request merged**